Algebraic_kernel_d/include/CGAL/Algebraic_kernel_d/Bitstream_coefficient_kernel_at_alpha.h

```diff
@@ -39,7 +39,7 @@ class Bitstream_coefficient_kernel_at_alpha_rep {
 
     typedef typename Algebraic_kernel_d_1::Algebraic_real_1 Algebraic_real_1;
 
-    Bitstream_coefficient_kernel_at_alpha_rep() {}
+    Bitstream_coefficient_kernel_at_alpha_rep() : _m_kernel(nullptr) {}
```

Comment:
Bit of a guess: maybe the compiler is imagining a default-constructed object is being used somewhere?

Reply:
Fixed it.